src/server/server.cc

```diff
@@ -1863,3 +1867,77 @@ std::string Server::GetKeyNameFromCursor(const std::string &cursor, CursorType c
 
   return {};
 }
+
+Status Server::AdjustWorkerThreadNum(int new_num) {
+  int now_num = (int)worker_threads_.size();
+  LOG(INFO) << __func__ << ", now_num: " << now_num << ", new_num: " << new_num;
+
+  if (now_num == new_num) {
+    return Status::OK();
+  } else if (now_num < new_num) {
```

Comment:
You delete `else` and write the rest of the logic without indentation due to an early return before.